Makefile

```diff
@@ -124,31 +126,40 @@ launch-dcos: check-terraform
 	cd .deploy; \
 	$(TERRAFORM_CMD) apply $(TERRAFORM_APPLY_ARGS) -var-file desired_cluster_profile
 
-.PHONY: kubectl-tunnel
-kubectl-tunnel:
-	$(KUBECTL_CMD) config set-cluster dcos-k8s --server=http://localhost:9000
-	$(KUBECTL_CMD) config set-context dcos-k8s --cluster=dcos-k8s --namespace=default
-	$(KUBECTL_CMD) config use-context dcos-k8s
-	$(call get_master_ip)
-	ssh -4 -o "UserKnownHostsFile=/dev/null" -o "StrictHostKeyChecking=no" -o "ServerAliveInterval=120" \
-		-N -L 9000:apiserver-insecure.kubernetes.l4lb.thisdcos.directory:9000 \
-		$(SSH_USER)@$(MASTER_IP)
+.PHONY: plan
+plan: plan-dcos
+
+.PHONY: deploy
+deploy: check-cli launch-dcos setup-cli install
+
+.PHONY: setup-cli
+setup-cli: check-dcos
+	@echo "Sleep for 15 seconds ..."
+	@sleep 15
```

Comment:
@rimusz why do we need to sleep?

Reply:
to wait for master be ready, specially it happens on was

Reply:
Is 15 seconds enough?